target/linux/bcm27xx/patches-6.1/950-0359-xhci-quirks-add-link-TRB-quirk-for-VL805.patch

```diff
@@ -22,17 +22,17 @@ Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
 
 --- a/drivers/usb/host/xhci-pci.c
 +++ b/drivers/usb/host/xhci-pci.c
-@@ -296,6 +296,7 @@ static void xhci_pci_quirks(struct devic
- 	if (pdev->vendor == PCI_VENDOR_ID_VIA && pdev->device == 0x3483) {
+@@ -295,6 +295,7 @@ static void xhci_pci_quirks(struct devic
+ 
+ 	if (pdev->vendor == PCI_VENDOR_ID_VIA && pdev->device == 0x3483)
  		xhci->quirks |= XHCI_LPM_SUPPORT;
- 		xhci->quirks |= XHCI_EP_CTX_BROKEN_DCS;
 +		xhci->quirks |= XHCI_AVOID_DQ_ON_LINK;
- 	}
```

Comment:
Parentheses are still necessary here.

Reply:
Would you mind making the needed change and pushing a commit to my branch so I can pull it please?

Reply:
Good catch. This is for the 6.1.43 commit.
The `if` needs curly brackets added `{`,`}` because there is more than one line to be run:
```
	if (pdev->vendor == PCI_VENDOR_ID_VIA && pdev->device == 0x3483) {
		xhci->quirks |= XHCI_LPM_SUPPORT;
		xhci->quirks |= XHCI_AVOID_DQ_ON_LINK;
	}
```
There are a few follow-up patches to the quirks for this device, so they will need refresh as well (your manually rebased list)

Reply:
Thanks it is clear to me now.

Reply:
Fixed, please verify.